kubernetes-client/pom.xml

```
@@ -220,7 +216,8 @@
                         META-INF/MANIFEST.MF,
                         META-INF/maven,
                         META-INF/maven/**/*,
-                        io/fabric8/kubernetes/client/dsl/internal/OperationSupport.*
+                        io/fabric8/kubernetes/client/dsl/internal/OperationSupport.*,
+                        io/fabric8/kubernetes/client/osgi/ManagedKubernetesClient.*
```

Maybe a more future proof expression?
```suggestion
                        io/fabric8/kubernetes/client/osgi/.*
```